README.md

```diff
@@ -1,50 +1,39 @@
-
-Itertools
-=========
+# Itertools
 
 Extra iterator adaptors, functions and macros.
 
-Please read the `API documentation here`__
-
-__ https://docs.rs/itertools/
-
-|build_status|_ |crates|_
+Please read the [API documentation here](https://docs.rs/itertools/)
 
-.. |build_status| image:: https://travis-ci.org/rust-itertools/itertools.svg?branch=master
-.. _build_status: https://travis-ci.org/rust-itertools/itertools
-
-.. |crates| image:: https://meritbadge.herokuapp.com/itertools
-.. _crates: https://crates.io/crates/itertools
+[![build_status](https://travis-ci.org/rust-itertools/itertools.svg?branch=master)](https://travis-ci.org/rust-itertools/itertools)
```

Comment:
This should probably be updated to use GitHub Actions instead of Travis CI.

Reply:
@atouchet Could you possibly review these patches and confirm that the new README.md resembles what we had previously in README.rst? If so, I'd merge it after your review.

Reply:
Sorry for missing this earlier. The changes look mostly good to me and I can submit some suggestions as a followup.